auto_examples/index.rst

```diff
@@ -0,0 +1,71 @@
+:orphan:
+
+.. _python-api-examples-index:
+
+Python API Examples
+===================
+
+This is the Flux Python API examples gallery that renders from our Python
+script examples!
+
+.. _api_examples:
+
+
+API Examples
+------------
+
+This gallery consists of introductory examples and general examples
+of using the Flux Python API.
+
+
+.. raw:: html
+
+    <div class="sphx-glr-thumbnails">
+
+
+.. raw:: html
+
+    <div class="sphx-glr-thumbcontainer" tooltip="This example will show how to submit a job using the Flux Python bindings.">
+
+.. only:: html
+
+  .. image:: /auto_examples/images/thumb/sphx_glr_example_job_submit_api_thumb.png
+    :alt: Introductory example - Job Submit API
+
+  :ref:`sphx_glr_auto_examples_example_job_submit_api.py`
+
+.. raw:: html
+
+      <div class="sphx-glr-thumbnail-title">Introductory example - Job Submit API</div>
+    </div>
+
+
+.. raw:: html
```

Comment:
is there a specific reason for selecting an example to look at with flux-logo image

![image](https://user-images.githubusercontent.com/274859/218793295-0b784c5c-c081-4451-825c-dd3cb4bbb4e7.png)



vs. just having a text listing, i.e.  

`Job submit API - introductory example, that will show you how to submit a job using the flux bindings`

b/c I imagine there may be a lot of examples eventually?

Reply:
I like the auto-generation with thumbnails better than without. I think it's fine if the same image is repeated/